statsd/localConfig.jsconfig

```diff
@@ -0,0 +1,8 @@
+{
+  graphitePort: 2003
+, graphiteHost: "localhost"
+, port: 8125
```

Comment:
On Mon, Nov 2, 2015 at 11:10 AM, Vinod Kurup <notifications@github.com>
wrote:

> Do we think there's value in making these ports pillar-configurable?


​Not until we have a need for it.​